packages/shared/postcss.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
       'postcss-simple-vars',
       {
         variables: {
-          pageMaxWidth: '40rem',
+          pageMaxWidth: '42.5rem',
```

Comment:
This might have app wide affects though, so maybe keep a close eye on production once deployed and run through some pages to ensure it's ok.

Reply:
Have made a check generally to see where this is used and see if it affected anything, from what I saw it affected the referral widget and account details which I already fixed. 
Will keep an eye anyways if something else shows up.